server.d.ts

```diff
@@ -1 +1 @@
-declare export function injectRSCPayload(rscStream: ReadableStream): TransformStream;
+declare export function injectRSCPayload(rscStream: ReadableStream, nonce?: string): TransformStream;
```

Comment:
Can we make this an options object with a nonce property? That way it's more extensible for more options in the future.

Reply:
Makes sense :+1:  Updated to `options?: { nonce?: string }`